honeybee_3dm/helper.py

```diff
@@ -26,3 +35,187 @@ def get_unit_system(file_3dm):
+        name: A text string of the name of the rhino object.
+        layer_name: A text string of the rhino layer name.
+
+    Returns:
+        A Honeybee Face object.
+    """
+    
+    obj_name = name or clean_and_id_string(layer_name)
+    args = [clean_string(obj_name), face_obj]
+    hb_face = Face(*args)
+    hb_face.display_name = args[0]
+    
+    if 'radiance_material' in config['layers'][layer_name]:
+        radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+        hb_face.properties.radiance.modifier = radiance_modifiers[config
+            ['layers'][layer_name]['radiance_material']]
+        return hb_face
+    else:
+        return hb_face
+
+
+def face3d_to_hb_object(config, face_obj, name, layer_name):
+    """Create Honeybee Aperture, Shade, and Door objects.
+
+    Args:
+        config: A dictionary of the config settings.
+        face_obj: A Ladybug Face3d object.
+        name: A text string of the name of the rhino object.
+        layer_name: A text string of the rhino layer name.
+
+    Returns:
+        A tuple of lists. Each list contains Honeybee Aperture objects, 
+            Honeybee Shade objects, and Honeybee Door objects. List will
+            be empty if no objects are found for that Honeybee object.
+    """
+
+    hb_shades, hb_apertures, hb_doors = tuple([[] for _ in range(3)])
+
+    obj_name = name or clean_and_id_string(layer_name)
+    args = [clean_string(obj_name), face_obj]
+    
+    if config['layers'][layer_name]['honeybee_face_object'] == 'aperture':
+        hb_aperture = Aperture(*args)
+        hb_aperture.display_name = args[0]
+        
+        if 'radiance_material' in config['layers'][layer_name]:
+            radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+            hb_aperture.properties.radiance.modifier = radiance_modifiers[config[
+                'layers'][layer_name]['radiance_material']]
+            hb_apertures.append(hb_aperture)
+        else:
+            hb_apertures.append(hb_aperture)
+    
+    elif config['layers'][layer_name]['honeybee_face_object'] == 'door':
+        hb_door = Door(*args)
+        hb_door.display_name = args[0]
+
+        if 'radiance_material' in config['layers'][layer_name]:
+            radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+            hb_door.properties.radiance.modifier = radiance_modifiers[config[
+                'layers'][layer_name]['radiance_material']]
+            hb_doors.append(hb_door)
+        else:
+            hb_doors.append(hb_door)
+
+    elif config['layers'][layer_name]['honeybee_face_object'] == 'shade':
+        hb_shade = Shade(*args)
+        hb_shade.display_name = args[0]
+
+        if 'radiance_material' in config['layers'][layer_name]:
+            radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+            hb_door.properties.radiance.modifier = radiance_modifiers[config[
```

Comment:
Reusable function will avoid mistakes like this one.

```suggestion
            hb_shade.properties.radiance.modifier = radiance_modifiers[config[
```